container_pipeline/lib/command.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+import subprocess
+
+
+def run_cmd(cmd, shell=False):
```

Comment:
A few reasons for this is:

- ``run_cmd`` will expand in the future, where we can have streaming output, etc.
- the structure of the ``lib`` pkg is that each of it's module contains a single functionality
- also, ``lib`` is something which is totally unaware of the pipeline, however, things in utils are kind of pipeline specific

I hope that this answers your questions.